test/lib/commands/audit.js

```diff
@@ -43,149 +33,130 @@ t.test('should audit using Arborist', t => {
     },
   })
 
-  const audit = new Audit(npm)
+  const npm = new Npm()
+  await npm.load()
+  npm.prefix = t.testdir()
 
   t.test('audit', async t => {
-    await audit.exec([])
-    t.match(ARB_ARGS, { audit: true, path: 'foo' })
+    await npm.exec('audit', [])
+    t.match(ARB_ARGS, { audit: true, path: npm.prefix })
     t.equal(AUDIT_CALLED, true, 'called audit')
     t.equal(AUDIT_REPORT_CALLED, true, 'called audit report')
-    t.equal(OUTPUT_CALLED, true, 'called audit report')
+    t.match(outputs, [['there are vulnerabilities']])
   })
 
   t.test('audit fix', async t => {
-    await audit.exec(['fix'])
+    await npm.exec('audit', ['fix'])
     t.equal(REIFY_FINISH_CALLED, true, 'called reify output')
   })
-
-  t.end()
 })
 
 t.test('should audit - json', async t => {
-  const npm = mockNpm({
-    prefix: 'foo',
-    config: {
-      json: true,
+  t.plan(1)
+  const { Npm } = mockNpm(t, {
+    'npm-audit-report': (_, opts) => {
+      t.match(opts.reporter, 'json')
```

Comment:
this test wasn't asserting anything?  At least this is better than nothing but the real fix is to introduce `nock` to our tests and use that instead. Assert the *actual* end-to-end behaviour we want.